macros/base/start_print.cfg

```diff
@@ -120,19 +118,15 @@ gcode:
     # 4 ----- GANTRY LEVELING ----------------------------------
     # Do a QGL, Z_TILT_ADJUST, or nothing depending of the machine configuration
     # The correct operation is automatically selected in the TILT_CALIBRATION macro
-    {% if status_leds_enabled %}
-        STATUS_LEDS COLOR="LEVELING"
-    {% endif %}
+    STATUS_LEDS COLOR="LEVELING"
 
     _TILT_CALIBRATE FORCE={force_homing_in_start_print}
 
 
     # 5 ----- EXTRUDER HEATING ---------------------------------
     # Heat the nozzle to print temperature ontop of the purge bucket
     # and wait 30s to let the nozzle dilate and stabilize
-    {% if status_leds_enabled %}
-        STATUS_LEDS COLOR="HEATING"
-    {% endif %}
```

Comment:
Can you put this test back ? it's mandatory to do it here...

As the file `status_leds.cfg` **is optionally included**, it's necessary to do the test **before** calling `STATUS_LEDS COLOR=xxx` to avoid an error about a non-existing macro.